### ISSUE TITLE
Feature request: Symbols

### ISSUE THREAD

**Issue by nightgolfer** [tom-james-watson/Emote#138]
**SYMBOLS**: I find it extremely convenient to have quick access to both emoji and special symbols via the same helper app; both are "characters" that don't have dedicated keyboard buttons, both live in text areas/boxes/documents. In combination with [Favorites](https://github.com/tom-james-watson/Emote/issues/137), Symbols give quick access to symbols that I use often, and I don't need to search for them. A few examples of Symbols added to Favorites on my Mac: ° ™ © ←→.

There are _a lot_ of Unicode characters. An idea could be to add a Symbols category button at the end of the emoji categories list, and populate it with a small subset of Unicode characters. In Settings, there could be an option for "Manage Unicode Sets", which opens a modal with a checkbox for each Unicode set/group. Adding a new set adds those characters to the Symbols category.

**Comment by tom-james-watson** [comment]
This has been suggested a couple of times before - https://github.com/tom-james-watson/Emote/issues/80 and https://github.com/tom-james-watson/Emote/issues/70.

I think maybe having a small subset is not a bad idea, honestly. This isn't something that is realistically going to happen soon, though. I'm not really actively developing the app at the moment.

**Comment by gesu92** [comment]
yeah a category with common unicode, like arrow, tables, misc symbols, maths,... or an unicode mode, this app it's so useful, thanks
